src/main/java/com/example/bookstore/repository/BookRepository.java

```diff
@@ -1,7 +1,20 @@
 package com.example.bookstore.repository;
 
 import com.example.bookstore.model.Book;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
```suggestion
@Repository
public interface BookRepository extends JpaRepository<Book, Long> {
```

Reply:
[This](https://stackoverflow.com/questions/65576247/why-put-repository-on-top-of-spring-data-jpa-interfaces) is what I found about annotation, so I guess I need to put it just for clarification?